### ISSUE TITLE
prompt is not always on top and can be lost behind window

### ISSUE THREAD

**Issue by blurymind** [p-sam/electron-prompt#11]
If you click on the main window, it will hide the prompt and you might lose it. You cant find it by alt+tab either.
Only way to get back to  it is to minimize the window

**Comment by p-sam** [comment]
Have you tried passing the main window as the parentWindow argument ? This would tie the prompt with the mainWindow

**Comment by blurymind** [comment]
I do:
```
const electron = require('electron')
const ipcRenderer = electron.ipcRenderer;
const remote = electron.remote, 
....
	prompt({
		title: 'Create New',
		label: 'Please enter the name:',
		value: 'new',
		parentWindow :remote.BrowserWindow,
		inputAttrs:{type: ''}}).then((newname) => {
			console.log('result', newname); // null if window was closed, or user clicked Cancel
			if (newname.length==0){alert("You didnt enter a name");return};
			createNew(newname);
			}).catch(console.error);
```

**Comment by blurymind** [comment]
when using
`parentWindow :remote.getCurrentWindow()`

I get 
`F:\stuff\elAPP\node_modules\electron-prompt\lib\index.js:38 Uncaught TypeError: Converting circular structure to JSON`
and the prompt doesnt even initialize properly

**Comment by blurymind** [comment]
Ok, now I am initiating it differently and not getting the error:
https://github.com/sperrichon/electron-prompt/issues/12#issuecomment-393560269
However it doesnt resolve any of the two problems.
I workaround I found to this issue is to set
https://github.com/sperrichon/electron-prompt/blob/master/lib/index.js#L14
to true. Its an ugly workaround, but at least the user cant lose it that way

**Comment by p-sam** [comment]
Did you find any solution other than set the window to be alwaysOnTop ? I can't think of any solution that doesn't make the api user having to handle the window behavior himself, and I would want to avoid that

**Comment by kickthedragon** [comment]
I made a pull request that fixes this issue.  You had a typo in your check for parent window that was passing a bool instead of the parent window object itself.

**Comment by p-sam** [comment]
Oh you're right it was introduced by PR #4 and I didn't see the bug

**Comment by p-sam** [comment]
Published/Released in 1.1.0 just now. I'm closing the issue, feel free to re-open if the bug was not completely resolved.
